polkadot/runtime/westend/src/lib.rs

```diff
@@ -2086,6 +2085,22 @@ parameter_types! {
 /// upgrades in case governance decides to do so. THE ORDER IS IMPORTANT.
 pub type Migrations = migrations::Unreleased;
 
+pub struct RenameAhClient;
+impl frame_support::traits::OnRuntimeUpgrade for RenameAhClient {
+	fn on_runtime_upgrade() -> Weight {
+		// CI-FAIL: set to the next spec of westend.
+		if VERSION.spec_version == 1_018_007 {
```

Comment:
```suggestion
		// CI-FAIL: set to the next spec of westend.
		if VERSION.spec_version == 1_018_011 {
```
Will set both to 1.18.11 as next version